scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -1140,7 +1141,7 @@ class FormatOps(
           Split(NoSplit.orNL(!shouldAddNewline), 0)
             .withIndent(indentParam, close2, ExpiresOn.Before)
         )
-      case Decision(t @ FormatToken(T.KwImplicit(), _, _), _)
+      case Decision(t @ FormatToken(T.KwImplicit() | T.Ident("using"), _, _), _)
```

Comment:
should we define a match like this in `util.TokenClasses` and reuse everywhere?

Reply:
We could have something similar to [SkUsing](https://github.com/scalameta/scalameta/blob/9a8781772b32a92fba6abd709d01ff328a20f34b/scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala#L848) in scalameta

Reply:
can we expose those `SkUsing` and others in scalameta, and just use them directly? they do multiple important things: check the dialect support and remove backticks, so that
```
Ident("using")
Ident("`using`")
```
would both work. it would be ideal to avoid duplication here, no?

Reply:
What do you think about https://github.com/scalameta/scalameta/pull/2193 ?